libraries/AP_HAL_ChibiOS/hwdef/ARKV6X/hwdef.dat

```diff
@@ -81,7 +85,8 @@ PD9 USART3_RX USART3
 
 # USART6 is for IOMCU
 PC6 USART6_TX USART6
-PC7 USART6_RX USART6
+# used for RC SBUS
+#PC7 USART6_RX USART6
```

Comment:
happy to bring this in, but I'm curious why SBUS didn't work on this UART? Did the user set SERIAL8_PROTOCOL to RCIN?

Reply:
Ping @dakejahl - unanswered question here.

Reply:
The conversation is in a now deleted slack workspace but I think ultimately the issue ended up being the user had bad hardware. I can test again to be sure. Are there concerns with this change?

Reply:
> The conversation is in a now deleted slack workspace but I think ultimately the issue ended up being the user had bad hardware. I can test again to be sure. Are there concerns with this change?

I think tridge was just curious.  He'll see your response as part of the backport process.

Reply:
I'm not as familiar with ardupilot so if something doesn't look right let me know. Thanks for the review